tests/test_status_service.py

```
@@ -23,23 +23,47 @@ def fake_ssl():
     assert "sekunder" in status["uptime"]
 
 
+def test_resolve_proxy_target_prefers_traefik_and_handles_invalid_port(monkeypatch, caplog):
+    monkeypatch.setenv("STATUS_NGINX_HOST", "nginx-service")
+    monkeypatch.setenv("STATUS_NGINX_PORT", "8081")
+    monkeypatch.setenv("STATUS_TRAEFIK_HOST", "traefik-service")
+    monkeypatch.setenv("STATUS_TRAEFIK_PORT", "notint")
+
+    captured = {}
+
+    def fake_check_tcp(host, port, timeout=2):
```

Ruff (0.14.14)

[warning] Unused function argument: `timeout`

(ARG001)